kustomize/v1alpha7/flatcar/kustomization.yaml

```diff
@@ -1,5 +1,7 @@
 resources:
 - ../default
 
-patchesStrategicMerge:
-- patch-flatcar.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
It is not technically anything wrong with having the `apiVersion` and `kind` in the middle of the file, but I would prefer that we keep it at the top (if specified at all). Now it looks a bit like it has something to do only with the `patches` :smile: